Resources/Prototypes/Entities/Objects/Fun/bike_horn.yml

```diff
@@ -39,3 +39,20 @@
     damage:
       types:
         Blunt: 0
+
+- type: entity
+  parent: BikeHorn
+  id: GoldenBikeHorn
+  name: golden honker
+  description: A happy honk prize, pray to the gods for your reward.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/goldbikehorn.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Fun/goldbikehorn.rsi
+    size: 5
+  - type: Clothing
+    sprite: Objects/Fun/goldbikehorn.rsi
+    slots: [Belt]
+  - type: Prayable
```

Comment:
does the horn still play a sound when you press z with this?

Reply:
yea it sitll plays a sound, luckily prayable is accessed via the right click menu and not immediate left click.

Reply:
yea it still honks with Z